src/coreComponents/physicsSolvers/wavePropagation/shared/WaveSolverBase.cpp

```diff
@@ -191,6 +185,18 @@ WaveSolverBase::WaveSolverBase( const std::string & name,
     setSizedFromParent( 0 ).
     setDescription( "Flag that indicates whether the receiver is local to this MPI rank" );
 
+  registerWrapper( viewKeyStruct::timestepStabilityLimitString(), &m_timestepStabilityLimit ).
```

Comment:
In fact in this PR we are closer to the logic of the non-linear solvers: if you put a forceDt too big for the CFL condition (or if you don't put it, as it will be putted at maxTime by default) the internal loop of the solver will do sub-steps to respect the CFL condition. 
For the moment, we decided to let this feature as an option to let the user be able to tests some dt on is own without getting into that routine. But we are not opposed to make it mandatory in the future.